docs/development_guide/README.md

```diff
@@ -89,11 +89,10 @@ This Open Service Mesh project uses [Go v1.19.0+](https://golang.org/). If you a
 
 ## Get the dependencies
 
-The OSM packages rely on many external Go libraries.
-
-Take a peek at the `go.mod` file in the root of this repository to see all dependencies.
+The OSM packages rely on many external Go libraries. Take a peek at the [go.mod](https://github.com/openservicemesh/osm/blob/main/go.mod) file in the root of this repository to see all dependencies.
 
 Run `go get -d ./...` to download all required Go packages.
```

Comment:
What is your go version and are you running these commands in your GOPATH?  

I thought running go get is no longer required with never versions of golang, but it might be becuase I don't use GOPATH:

```
go help gopath-get
The 'go get' command changes behavior depending on whether the
go command is running in module-aware mode or legacy GOPATH mode.
```

  should this maybe updated to `go mod tidy`? which shouldn't require setting `GO111MODULE`. 

I think since 1.17+ `GO111MODULE=on` is the default behavior. In my local env I don't have it set:

```
❯ go env
GO111MODULE=""
```

Reply:
My go version is : `go version go1.19.1 darwin/amd64` . I think this is the latest version. Interestingly, `GO111MODULE` is `off` by default. I am not sure this is because I installed go using Homebrew.

Reply:
is it `off` or empty?  If it is `off` I would be surprised but could be something to do with the homebrew installation